presto-cache/src/main/java/com/facebook/presto/cache/alluxio/AlluxioCacheConfig.java

```diff
@@ -191,4 +192,17 @@ public AlluxioCacheConfig setCacheQuotaEnabled(boolean cacheQuotaEnabled)
         this.cacheQuotaEnabled = cacheQuotaEnabled;
         return this;
     }
+
+    public boolean isCacheKeyModificationTimeIncluded()
+    {
+        return cacheKeyModificationTimeIncluded;
+    }
+
+    @Config("cache.alluxio.cache-key.modification.included")
```

Comment:
to be consistent, how about cache.alluxio.cache-key.modification-time.included ?